rpcs3/Emu/RSX/gcm_printing.cpp

```diff
@@ -853,13 +854,13 @@ namespace
 		{NV3089_IMAGE_IN_FORMAT, "NV3089_IMAGE_IN_FORMAT"},
 		{NV3089_IMAGE_IN_OFFSET, "NV3089_IMAGE_IN_OFFSET"},
 		{NV3089_IMAGE_IN, "NV3089_IMAGE_IN"},
-		{GCM_SET_DRIVER_OBJECT, "GCM_SET_DRIVER_OBJECT"},
-		{GCM_DRIVER_QUEUE, "GCM_DRIVER_QUEUE + 0x0"},
-		{GCM_DRIVER_QUEUE+1, "GCM_DRIVER_QUEUE + 0x4"},
-		{GCM_FLIP_HEAD, "GCM_FLIP_HEAD"},
-		{GCM_FLIP_HEAD+1, "GCM_FLIP_HEAD + 0x4"},
-		{GCM_SET_USER_COMMAND, "GCM_SET_USER_COMMAND"},
-		{GCM_FLIP_COMMAND, "GCM_FLIP_COMMAND"},
+		{GCM_SET_DRIVER_OBJECT, "SET_DRIVER_OBJECT"},
```

Comment:
Why are these renamed

Reply:
GCM_ prefix is redundent for all methods here because it is added elsewhere.